src/formatDate.js

```diff
@@ -8,7 +8,75 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  // create property for symbols from and to
+  const simbolFormat = fromFormat[fromFormat.length - 1];
+  const simbolTo = toFormat[toFormat.length - 1];
```

Comment:
Similarly, 'simbolTo' should be renamed to 'symbolTo' to correct the typo and maintain consistency.